index.html

```diff
@@ -68,9 +68,10 @@ <h2 class="title">Magento APIs</h2>
       <img src="{{ site.baseurl }}/i/icons/community.svg" alt="" />
       <h2 class="title">Community</h2>
       <ul>
-          <li><a href="{{ site.baseurl }}/community/resources">Community Resources</a></li>
-        <li><a href="{{ page.baseurl }}/contributor-guide/contributing_dod.html">Magento Definition of Done</a></li>
         <li><a href="{{ page.baseurl }}/contributor-guide/contributing.html">Contributor Guide</a></li>
+        <li><a href="{{ page.baseurl }}/contributor-guide/contributing_dod.html">Magento Definition of Done</a></li>
+        <li><a href="{{ site.baseurl }}/community/resources/docs-as-code.html">Docs as Code</a></li>
```

Comment:
Don't use this term to describe the new concept we're introducing; "docs as code" already has a meaning in our industry.

Reply:
Renaming, see previous comment.